From 33c8db58c53ebf785fabc6070a567b23c02355c5 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 30 Aug 2018 08:48:18 +0800
Subject: [PATCH 4481/5242] MLK-19395 ARM64: dts: freescale:
 imx8qxp-ddr3l-val: correct model name

commit  8534e889d7e12bded6b19aa692052b6c5e635f83 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8QXP DDR3L board includes i.MX8QXP LPDDR4 ARM2 board's dts,
but need to overwrite the model name accordingly.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-ddr3l-val.dts   |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-ddr3l-val.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-ddr3l-val.dts
index c9a39e8..3e52899 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-ddr3l-val.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-ddr3l-val.dts
@@ -15,6 +15,9 @@
 #include "fsl-imx8qxp-lpddr4-arm2.dts"
 
 / {
+	model = "Freescale i.MX8QXP DDR3L VALIDATION";
+	compatible = "fsl,imx8qxp-ddr3l-val", "fsl,imx8qxp";
+
 	reserved-memory {
 		linux,cma {
 			compatible = "shared-dma-pool";
-- 
1.7.9.5

